Provide sample python function (#377)
kyma-project/serverless-manager: examples/python-text2img/README.md

# Overview

> The function uses [Deep AI API](https://deepai.org/machine-learning-model/text2img) for image generation from text.

Reviewer: ```suggestion
The Function uses [Deep AI API](https://deepai.org/machine-learning-model/text2img) for image generation from the text.
```

> It's purpose is to demonstrate how to achieve basic development tasks as a python function developer:

Reviewer: ```suggestion
Its purpose is to demonstrate how to achieve basic development tasks as a Python Function developer, such as:
```

>  - How to consume request

Reviewer: ```suggestion
 - How to consume a request
```

>  - How to customise response

Reviewer: ```suggestion
 - How to customize a response
```

 - How to read ENVs
 - How to configure libraries as dependencies

>  - How to send requests from the function code

Reviewer: ```suggestion
 - How to send requests from the Function code
```

>  - How to configure function in the Kyma context

Reviewer: ```suggestion
 - How to configure Function in the Kyma context
```

   - function API exposure
   - function ENVs injection

> ## Deploy function

Reviewer: ```suggestion
## Deploy Function
```

> Copy the `resources/secrets/deepai-template.env` into `resources/secrets/deepai.env` and fill in `API-KEY`.

Reviewer: ```suggestion
1. Copy `resources/secrets/deepai-template.env` into `resources/secrets/deepai.env` and fill in `API-KEY`.
```

> Then, deploy the application via

Reviewer: ```suggestion
2. Deploy the application.
```
